projects/rocprofiler-systems/docker/Dockerfile.opensuse

```
@@ -25,7 +25,7 @@ RUN zypper --non-interactive update -y && \
     zypper --non-interactive install -y -t pattern devel_basis && \
     zypper --non-interactive install -y binutils-gold chrpath cmake curl dpkg-devel \
         gcc-c++ git iproute2 libdrm-devel libnuma-devel ninja openmpi3-devel python3-pip rpm-build \
-        sqlite3-devel wget  && \
+        sqlite3-devel wget nlohmann_json-devel && \
```

```suggestion
        gcc-c++ git iproute2 libdrm-devel libnuma-devel nlohmann_json-devel ninja \
        openmpi3-devel python3-pip rpm-build sqlite3-devel wget && \
```
Sorting list alphabetically.